src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {

```

Comment:
The class name 'ColorSupplier' is a bit misleading as it suggests that it is supplying multiple colors or is a utility for supplying colors. Consider renaming it to something like 'RandomColorGenerator' to better reflect its purpose of generating a random color.